vignettes/layer.Rmd

```diff
@@ -16,4 +16,301 @@ knitr::opts_chunk$set(
 
 ```{r setup}
 library(tlang)
+library(magrittr)
+```
+
+A core design element of `tlang` was to offer the ability to layer and build template table formats that could be shared, improved, and eventually customized for a specific table. The idea being that organizations could develop a standard `tfrmt` object that would serve as the base format for the rest of the tables, and make it so programmers could focus on setting up the data as opposed to formatting of the table. 
+
+There are two ways that `tfrmt` objects may be layered. The preferred method is through the first argument of `tfrmt` - `tfrmt_obj`. The second, but perhaps more explicit method is using the `layer_tfrmt` function. We will go through their application below.  
+
+## Creating a template
+
+Before we show how to layer a tfrmt, we need to create a template tfrmt. A template is intended to provide the basics values that an organization would expect to be repeated across multiple tables. For example, in a table such as an Adverse Events table, there are some standards that can be expected that allows us to constructing a basic `tfrmt`:
+
+First, we know that the cells should be consistently formatted as follows, which allows us to create a body_plan:
+
+  - Total counts to be whole numbers 
+  - Cells under each treatment to be a combination of a count and percent of that population. When percent is 100 or zero, value should be blank.
+  - p-values to be formatted where: >.99 is displayed as ">0.99", <0.001 is displayed as "<0.001", and any values between should be displayed to three decimal places. Missing p-values should be replaced with "--".
+  
+Next, we can apply standard to our organizations AE ARD format to ensure that names of are set consistently:
```

Comment:
```suggestion
Next, because our data is standardized as an analysis results dataset, we can pre-fill the expected column names. This also ensures that the data abides to the standard and names are set consistently:
```